pango/src/auto/attr_list.rs

```diff
@@ -86,10 +97,26 @@ impl AttrList {
             ffi::pango_attr_list_update(self.to_glib_none().0, pos, remove, add);
         }
     }
+
+    #[cfg(any(feature = "v1_50", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+    #[doc(alias = "pango_attr_list_from_string")]
+    pub fn from_string(text: &str) -> Option<AttrList> {
```

Comment:
Should probably be a `Result`

Reply:
And implement `FromStr`?

Reply:
I am pretty sure I did this at some point...meh, got some changes lost during the rebase

Reply:
Never mind, I did that for TabArray. Will do!